src/docs/development/add-to-app/android/add-flutter-fragment.md

```diff
@@ -145,12 +145,11 @@ class MyActivity : FragmentActivity() {
 ```
 {% endsamplecode %}
 
-The above code is sufficient to render a Flutter UI that begins with a call to
+The previous code is sufficient to render a Flutter UI that begins with a call to
```

Comment:
would above be clearer here since it's a very spacial reference. Previous may refer to the code the user already had (from the create template or some such).